src/stancjs/stancjs.ml

```diff
@@ -72,21 +71,26 @@ let stan2cpp model_name model_string flags =
                Pedantic_analysis.print_warn_uninitialized mir ;
              if is_flag_set "warn-pedantic" then
                Pedantic_analysis.print_warn_pedantic mir ;
-             (cpp, []) )
+             cpp )
+
+let stanc_stdout = ref ""
+let stanc_stderr = ref ""
```

Comment:
Would it be better to use a list or something?

Reply:
yeah, a list would be constant time to append to instead of O(N)